tools/Makefile.unix

```diff
@@ -697,3 +704,8 @@ apps_distclean:
 ifneq ($(APPDIR),)
 	$(Q) $(MAKE) -C "$(TOPDIR)/$(APPDIR)" TOPDIR="$(TOPDIR)" distclean
 endif
+
+check_format:
+	$(call colorecho,'Checking NuttX formatting with nxstyle')
+	@$(TOPDIR)/tools/check_code_style.sh
+	@cd "$(SRC_DIR)" && git diff --check
```

Comment:
There is another problem here:  You must account for that fact that .git/ is NOT present in most peoples environment.  Please make the last line conditioned on #ifeq ($(GIT_DIR),y) or you will break 80% of the nuttx user's builds!

Reply:
@patacongo see [here](https://github.com/apache/incubator-nuttx/pull/2#discussion_r360727667)

Reply:
The simplest solution would be to make the whole check_format target available only ifeq ($(GIT_DIR),y).  that would only be conditional logic at 708 and 195.  Pretty easy.

Reply:
Yeah I already had thought of that. I had time to think - but not change it. The days have been very, very long here with the added overhead of ASF.  

I would like to ask you to contribute. It is far better for building community to add value and help with what you know. 

Understanding that using Github is new to you. I woulds suggest you take these steps to help. It will reduce the back and forth. it is like ifdef rash in code - we should avoid it.

- Be patient - we all that other commitments not just NuttX. Theres is now way for a team to move as quickly as you did on you own. 

- Use the [Suggesting Changes on GitHub](https://haacked.com/archive/2019/06/03/suggested-changes/) This makes your comments the solution as works well for the [manini](https://en.wiktionary.org/wiki/manini) stuff.
- Lean to use the mark up.  You can say [ditto](https://github.com/apache/incubator-nuttx/pull/2/files#r360726623) And avoid a messy review. 
You can then link to the NuttX sites coding  standard instead of re typing them all the time.